plugins/sql-language/src/main/java/org/polypheny/db/sql/language/validate/IdentifierNamespace.java

```diff
@@ -109,14 +127,15 @@ private SqlValidatorNamespace resolveImpl( SqlIdentifier id ) {
         SqlValidatorScope.Resolve previousResolve = null;
         if ( resolved.count() == 1 ) {
             final SqlValidatorScope.Resolve resolve = previousResolve = resolved.only();
-            if ( resolve.remainingNames.isEmpty() ) {
+            return resolve.namespace;
+            /*if ( resolve.remainingNames.isEmpty() ) {
                 return resolve.namespace;
-            }
+            }*/
             // If we're not case-sensitive, give an error.
             // If we're case-sensitive, we'll shortly try again and give an error then.
-            if ( !nameMatcher.isCaseSensitive() ) {
+            /*if ( !nameMatcher.isCaseSensitive() ) {
                 throw validator.newValidationError( id, Static.RESOURCE.objectNotFoundWithin( resolve.remainingNames.get( 0 ), SqlIdentifier.getString( resolve.path.stepNames() ) ) );
-            }
+            }*/
```

Comment:
What is going on here? Shouldn't this be the job of the snapshot?

Reply:
Artifact and not longer used, removed.